packages/next/client/index.js

```diff
@@ -520,10 +520,24 @@ async function doRender({ App, Component, props, err }) {
     appProps,
   })
 
+  if (lastRenderResolver) {
+    lastRenderResolver()
```

Comment:
Wouldn't this fire `after-reactdom-render` requests for canceled renders? Is that OK? Might also be good to note in comments.